cmd/build.go

```diff
@@ -261,5 +269,11 @@ ex) git@github.com:<org>/<repo>.git[#<branch>]
 		false,
 		"Use sudo command for container runtime.")
 
+	buildCmd.Flags().BoolVar(
+		&useLocalConfig,
+		"local",
+		false,
+		"Run command with .sdlocal/config file in current directory.")
+
```

Comment:
Which is better to declare `--local` option at global or at each subcommands?

Reply:
`--local` option is only used by `build` and `config` subcommand now.
If something using `--local` option is added, then we should rethink about it.

Reply:
I see.